js/views/settingsVw.js

```diff
@@ -774,6 +777,33 @@ module.exports = Backbone.View.extend({
     });
   },
 
+  launchServerConfig: function() {
+    var serverConnectModal = new ServerConnectModal({
+      includeCloseButton: true
+    }).render().open();
+
+    this.serverConnectSyncHandler &&
+      this.stopListening(app.serverConfig, null, this.serverConnectSyncHandler);
+
+    this.serverConnectSyncHandler = function() {
+      // todo: bit of a hack to hide the close button. really, the
+      // modal api should provide this if we want to allow
+      // this type of stuff.
```

Comment:
We probably do want to put this in the modal api, I'm sure there will be other situations where we want to deactivate the close button.